File: Estatistica I/amostragem_simples.r
#amostragem simples

amostra = sample(c(0,1), 150, replace = TRUE, prob = c(0.5, 0.5)) #gera uma amostra simples com 150 numeros entre 0 e um com reposicao e probablidade de 0.5 em cada 
length((amostra[amostra==0])) #quantidade de 0 na variavel amostra
length((amostra[amostra==1])) #quantidade de 1 na variavel amostra
  
set.seed(1234) #define uma semente de aleatoriedade
sample(c(100), 1) #gera um numero ate 100 (sempre o mesmo pela seed)

File: Estatistica I/amostragem_simples.py
#amostragem simples

import pandas as pd #importa o pandas como pd
import numpy as np #importa o numpy como np

base = pd.read_csv('iris.csv') #le o iris.csv com uma funçao do pandas e guarda na variavel

base.shape #retorna as medidas do objeto

np.random.seed(1234) #usa o numpy para travar a semente de aleatoriedade
amostra = np.random.choice(a = [0, 1], size = 150, replace = True, p = [0.5, 0.5]) #gera 150 numeros aleatorios entre 0 e 1 com reposicao e probabilidade de 0.5 cada

len(amostra) #retorna o tamanbo da amostra
len(amostra[amostra==0]) #retorna a quantidade de 0 em amostra
len(amostra[amostra==1]) #retorna a quantidade de 1 em amostra